.bazelrc

```diff
@@ -19,7 +19,6 @@ build --incompatible_default_to_explicit_init_py
 
 # Windows makes use of runfiles for some rules
 build --enable_runfiles
-startup --windows_enable_symlinks
```

Comment:
For users do have admin right, turning on `--windows_enable_symlinks` could be a good thing to improve efficiency, maybe document that somewhere?